### PR TITLE
Map and Searchbox

### PR DESCRIPTION
**Pull request by BernabeFelix**
- Add map in home as example, this will be moved to promo detail
- Add SearchBox autocomplete as CustomForm to be used anywhere

### CODE REVIEW

File: client/src/components/CustomerAdmin/AdminEditPromotion/controls.js [hunk 1]
```diff
@@ -20,6 +19,16 @@ const controls = {
       type: Input.text,
       floatingLabelText: 'descripción'
     }
+  },
+  title: {
+    errors: {
+      required: 'Ingrese un titulo'
```
Comment (paulsancer) [inline client/src/components/CustomerAdmin/AdminEditPromotion/controls.js]: `título`  I think we should find some tool for spelling checks.

File: client/src/components/CustomerAdmin/NewEditPromotion.jsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import controls from './AdminEditPromotion/controls';
 import Form from '../common/Form';
 import CustomTextField from '../Auth/CustomFormField/CustomTextField';
 import { NewPromotion } from '../common/types';
+import SearchBox from '../Map/SearchBox';
 
 // const dropZoneStyle = {
```
Comment (paulsancer) [inline client/src/components/CustomerAdmin/NewEditPromotion.jsx]: Will we need this?

File: client/src/components/CustomerAdmin/AdminEditPromotion/controls.js [hunk 3]
```diff
@@ -20,6 +19,16 @@ const controls = {
...
+    },
+    fields: {
+      name: 'title',
+      type: Input.text,
+      floatingLabelText: 'titulo'
```
Comment (paulsancer) [inline client/src/components/CustomerAdmin/AdminEditPromotion/controls.js]: `título`

File: client/src/components/Map/Marker.jsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+const Marker = ({ title }) => (
+  <div
+    style={{
+      width: '30px',
+      color: 'black',
+      height: '30px',
+      fontSize: '18px',
+      borderRadius: '50%',
+      backgroundColor: 'green'
```
Comment (paulsancer) [inline client/src/components/Map/Marker.jsx:12]: Is this green final? We could move `green` and `black` into the `variables.js` file

File: client/src/components/Auth/CustomFormField/CustomField.jsx [hunk 5]
```diff
@@ -86,6 +86,7 @@ class CustomField extends Component {
     }
   };
 
+  // todo: delete this reset if not used
```
Comment (paulsancer) [inline client/src/components/Auth/CustomFormField/CustomField.jsx]: We don't need this anymore, please get rid of it

**Review by paulsancer — CHANGES_REQUESTED** [review]
Small changes, if not applicable, go ahead and merge

File: client/src/components/CustomerAdmin/AdminEditPromotion/controls.js (hunk 1 above)
Comment (BernabeFelix) [reply client/src/components/CustomerAdmin/AdminEditPromotion/controls.js]: thanks. We should not spend time in this, spell checking is not development, is copy and is not been provided by our client. We should only worry about copied given.

File: client/src/components/CustomerAdmin/NewEditPromotion.jsx (hunk 2 above)
Comment (BernabeFelix) [reply client/src/components/CustomerAdmin/NewEditPromotion.jsx]: yes we will, I'll add a ToDo

File: client/src/components/Map/Marker.jsx (hunk 4 above)
Comment (BernabeFelix) [reply client/src/components/Map/Marker.jsx:12]: is not, we are waiting for instructions

**Comment by BernabeFelix** [comment]
All feedback was implement, I'll merge for integration with database purposes

**Pull request merged**